src/frontend/e2e/setup/local.config.ts

```diff
@@ -1,8 +1,11 @@
 import { PlaywrightTestConfig } from "@playwright/test";
 import { devices } from "@playwright/test";
 import dotenv from "dotenv";
+import path from "path";
 
-dotenv.config();
+dotenv.config({
+  path: path.resolve(__dirname, "../../", `.env.${process.env.NODE_ENV}`),
+});
```

Comment:
why do we need this now?

Reply:
I have hard coded the file name since this environment specific config